src/extractors/custom/www.asahi.com/index.js

```diff
@@ -0,0 +1,33 @@
+export const WwwAsahiComExtractor = {
+  domain: 'www.asahi.com',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
+  },
+
+  author: {
+    selectors: [['meta[name="article:author"]', 'value']],
+  },
+
+  date_published: {
+    selectors: [['meta[name="pubdate"]', 'value']],
+  },
+
+  dek: {
+    selectors: [['meta[name="og:description"]', 'value']],
```

Comment:
I have noticed that on this website, the `og:description` is automatically generated as the first few words of the article content, as opposed to a sub-headline or special introductory text. This selector in this case is more suited as an excerpt rather than a dek.

Reply:
Thank you. Fixed.